.github/workflows/ping-codeowners-issues.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Ping code owners on issues'
```

Comment:
Is this essentially a rename that calls the script that was renamed?

Reply:
Yes. The one change is that I did [remove the label filter](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16175/files?short_path=fb0fd06#diff-fb0fd06da4d79a15e0d66efd4c1c24e84ad41d37b152f4bcb7ea1e218d547072L8) to run the workflow. I'm open to re-adding it if we want it, but I figure it's relatively a minor inefficiency for not having to maintain the list of component types.

Reply:
Ya I'm good with that